third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -3963,6 +3972,13 @@ void WebMediaPlayerImpl::UnregisterFrameSinkHierarchy() {
     bridge_->UnregisterFrameSinkHierarchy();
 }
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
```

Comment:
Same for here. Combine this `USE_STARBOARD_MEDIA` block with `WebMediaPlayerImpl::GetSetBoundsCB()`.